README.md

```diff
@@ -69,12 +69,15 @@ and a [SPARQL endpoint](http://www.w3.org/TR/sparql11-protocol/) as sources:
 ```
 
 The following sources are supported out of the box:
-- HDT files ([`HdtDatasource`](https://github.com/LinkedDataFragments/Server.js/blob/master/lib/datasources/HdtDatasource.js) with `file` setting)
+
+- HDT files ([`HdtDatasource`](https://github.com/LinkedDataFragments/Server.js/blob/master/lib/datasources/HdtDatasource.js) with `file` setting). Use `ExternalHdtDatasource` (or setting `external` to `true`) for large data sets.
```

Comment:
To be precise: `external` is useful when you have _many_ large datasets in a single instance. Specifically, it was added for the LOD Laundromat, which has hundreds of thousands of HDT datasources in one endpoint. If you just have a (couple of) dozen HDT files, it won't be a problem at all. In general, first try without `external`, and only add it if you run out of system memory.

Reply:
Maybe we can also make explicit that this is the recommended backend.

Reply:
Sorry, I don't understand the need of `ExternalHdtDatasource` and its particular difference to `HdtDatasource`. Anyway, recommendation to use SPARQL or HDT for large data sets should be enough.

Reply:
HdtDatasource means loading an HDT file in-process through https://github.com/RubenVerborgh/HDT-Node/. ExternalHdtSource means spawning an external process to read the HDT file. In the first case, the HDT file remains loaded (as a memory-mapped file) for the duration of the server process. In the second case, the HDT file is loaded for the duration of the request. The second uses less memory, but is slower because of the reloading.

Reply:
Thanks I've updated the pull request.